recipes/example/meta.yaml

```diff
@@ -1,83 +1,23 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
-
-# Jinja variables help maintain the recipe as you'll update the version only here.
-# Using the name variable with the URL in line 13 is convenient
-# when copying and pasting from another recipe, but not really needed.
-{% set name = "simplejson" %}
-{% set version = "3.8.2" %}
+{% set version = "2.2.203" %}
+{% set sha256 = "d1e2368b1335a6a5f496b887950ad5da3d85783ec76f74a663214989817bb497" %}
 
 package:
-  name: {{ name|lower }}
+  name: dotnet
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  # If getting the source from GitHub remove the line above
-  # uncomment the line below and modify as needed
-  # url: https://github.com/simplejson/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f
-  # sha256 is the prefered checksum -- you can get it for a file with:
-  #  `openssl sha256 <file name>`.
-  # You may need the openssl package, available on conda-forge
-  #  `conda install openssl -c conda-forge``
+  url: https://download.visualstudio.microsoft.com/download/pr/647f8505-3bf0-48c5-ac0f-3839be6816d7/d0c2762ded5a1ded3c79b1e495e43b7c/dotnet-sdk-{{ version }}-linux-x64.tar.gz
```

Comment:
May as well support all platforms:
```
{% set platform = "linux" %}  # [linux]
{% set platform = "osx" %}    # [osx]
{% set platform = "win" %}    # [win]
{% set ext= "tar.gz" %}       # [not win]
{% set ext= "zip" %}          # [win]
```
```yaml
    url: https://dotnetcli.azureedge.net/dotnet/Sdk/{{ version }}/dotnet-sdk-{{ version }}-{{ platform }}-x64.{{ ext }}
```
I think the above is a better url to use (pulled from [`dotnet-install.ps1 `](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script))